src/libstd/sys/unix/c.rs

```diff
@@ -15,11 +15,26 @@
 
 pub use self::select::fd_set;
 pub use self::signal::{sigaction, siginfo, sigset_t};
+#[cfg(not(target_os = "nacl"))]
 pub use self::signal::{SA_ONSTACK, SA_RESTART, SA_RESETHAND, SA_NOCLDSTOP};
+#[cfg(not(target_os = "nacl"))]
 pub use self::signal::{SA_NODEFER, SA_NOCLDWAIT, SA_SIGINFO, SIGCHLD};
 
+#[cfg(target_os = "nacl")]
+pub use self::signal::{SA_NOCLDSTOP, SA_SIGINFO};
+
 use libc;
 
+// For (P)NaCl targets, nacl_io is a Pepper wrapper providing some C functions
+// missing in Newlib/libnacl.
+#[cfg(all(target_os = "nacl", not(test)))]
+#[link(name = "nacl_io", kind = "static")] extern {}
```

Comment:
What functions is this library providing? I think it'd make more sense to put this in `rtdeps.rs` instead of here (same for `c++` below)